test/unittests/commands/xmlupload/test_transform_file_values.py

```diff
@@ -442,32 +437,5 @@ def test_make_iiif_uri_value_serialised() -> None:
     assert serialised == expected
 
 
-def test_make_boolean_value_with_permissions() -> None:
```

Comment:
This is now tested together with all the other values. This file now only contains file value transformations